graphql/schemas/Social/channels.graphql

```diff
@@ -0,0 +1,46 @@
+type SocialChannel {
+    id: ID!
+    name: String!
+    slug: String
+    description: String!
+    entity_namespace: String!
+    entity_id: ID!
+    last_message_id: ID!
+    messages: [Messages]
```

Comment:
@FredPeal list have to follow the standard [Message!]! @hasMany

Reply:
That is a relationship many to many, @hasMany correspond to one has many relationships